docker/Dockerfile.olbase

```diff
@@ -31,6 +29,9 @@ RUN apt-get -qq update && apt-get install -y \
     parallel
 
 # Print gnu parallel citation
+USER openlibrary
+RUN echo 'will cite' | parallel --citation
+USER root
 RUN echo 'will cite' | parallel --citation
```

Comment:
Do we ___need both___ users to commit to the citation?

Reply:
:D Haha, apparently :P That way when you use `parallel` as root (which the Dockerfile does during build) or as a person via `home`, you won't see the little citation marker.